### DISCUSSION TITLE
How do you get the Galaxy Brain Badge?

### DISCUSSION THREAD

**Discussion by Nateay0** [community/community#30600] [Other Feature Feedback, Questions, & Ideas] [answered]
How?

**Comment by felipezanardi** [comment, marked as answer]
You need to answer at least 2 discussions.
Also, you can't mark your own answers to your own questions, they have to be marked by another user.
If you wanna know more about the other badges, check this: https://github.com/Schweinepriester/github-profile-achievements

**Reply by Nateay0** [reply]
Thank you

**Comment by github-staff** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.

**Comment by Pavel-Shcherbo** [comment]
Response #2

**Comment by Hazrat-Ali9** [comment]
How do I get a Galaxy Brain badge
Response
